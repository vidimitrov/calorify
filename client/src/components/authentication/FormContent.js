import styled from 'styled-components';

const FormContent = styled.div`
  -webkit-border-radius: 10px 10px 10px 10px;
  border-radius: 10px 10px 10px 10px;
  background: #fff;
  padding: 30px;
  width: 90%;
  max-width: 450px;
  position: relative;
  padding: 0px;
  -webkit-box-shadow: 0 30px 60px 0 rgba(0,0,0,0.3);
  box-shadow: 0 30px 60px 0 rgba(0,0,0,0.3);
  text-align: center;
`;

export default FormContent;
