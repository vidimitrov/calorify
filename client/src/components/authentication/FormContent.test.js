import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import FormContent from './FormContent';

it('Authentication/FormContent component renders correctly', () => {
  const tree = renderer.create(<FormContent>Some content</FormContent>).toJSON();
  expect(tree).toMatchSnapshot();
});
