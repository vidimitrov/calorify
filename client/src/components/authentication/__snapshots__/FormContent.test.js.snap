// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authentication/FormContent component renders correctly 1`] = `
.c0 {
  -webkit-border-radius: 10px 10px 10px 10px;
  border-radius: 10px 10px 10px 10px;
  background: #fff;
  padding: 30px;
  width: 90%;
  max-width: 450px;
  position: relative;
  padding: 0px;
  -webkit-box-shadow: 0 30px 60px 0 rgba(0,0,0,0.3);
  box-shadow: 0 30px 60px 0 rgba(0,0,0,0.3);
  text-align: center;
}

@media (max-width:420px) {
  .c0 {
    width: 100%;
  }
}

<div
  className="c0"
>
  Some content
</div>
`;
